english/BayesTheorem/priors/chocolateOrSleep.md

```diff
@@ -0,0 +1 @@
+Would you like some more [chocolate](../../search_for_chocolate/search_for_chocolate.md) or would you like to go to [sing](../../sing/sing.md)?
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.